config/common.mk

```diff
@@ -123,7 +123,11 @@ PRODUCT_PACKAGES += \
     themed_bootanimation
 
 # GApps
+ifeq ($(NO_GAPPS_BUILD),true)
+#include vendor/gapps/config.mk
+else
 include vendor/gapps/config.mk
+endif
 
```

Comment:
Should be 
`ifneq($(NO_GAPPS_BUILD),true)
include vendor/gapps/config.mk
endif`